client/webserver/site/src/js/wallets.ts

```diff
@@ -636,8 +636,11 @@ export default class WalletsPage extends BasePage {
 
       const walletDef = app().walletDefinition(assetID, wallet.type)
       page.walletType.textContent = walletDef.tab
+      Doc.show(page.reconfigureBox)
```

Comment:
I don't think you need to explicitly show this. Just remove `page.reconfigureBox` from the `Doc.hide` above.